ext/pgsql/pgsql.c

```diff
@@ -5419,15 +5419,15 @@ static int php_pgsql_fd_cast(php_stream *stream, int cast_as, void **ret) /* {{{
 		case PHP_STREAM_AS_FD_FOR_SELECT:
 		case PHP_STREAM_AS_FD:
 		case PHP_STREAM_AS_SOCKETD:
```

Comment:
Needs a code block.

Reply:
Oh, right, thanks! I could drop that for 8.0+, couldn't I?

Reply:
I believe this requirement still exists in C99.

Reply:
Seems you're right.